README.md

```diff
@@ -29,7 +29,7 @@
 
 # 📖 Movie App <a name="about-project"></a>
 
-**Movie App**  is about building a mobile web application to check a list of metrics (numeric values) that you will create making use of React and Redux. The application will allow users to search for appropriate choice movie. Application is working with the real live data from API.
+**Movie App**  is about building a mobile web application to check a list of metrics (numeric values) that you will create making use of React and Redux. The application will allow users to search for appropriate choice movie to view. Application is working with the real live data from API.
 
 # 👁‍🗨 Preview
 
```

Comment:
- [ ] Please the way you present the details on the details page doesn't match the design also the font color .

| Your code  | Behance |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/74744787/212478997-d19b1a70-5b93-45a3-8b5b-75a7fcad62e4.png)  | ![image](https://user-images.githubusercontent.com/74744787/212479245-17d5ff7b-8e28-4b4e-b38c-58c03559efb0.png) |